docs/services/social_providers.md

```diff
@@ -21,8 +21,8 @@ If you want to enable sign up through social providers like Facebook you will ne
 3. Click the "Create New App" button.
 4. Fill in the `Name`, `Description` fields.
 5. Fill in the `Website` and `Callback URL` fields with the same value. If you are working on a development app you need to use `http://127.0.0.1:3000/` instead of `http://localhost:3000/`.
-5. Check the 'Developer Agreement' checkbox and click the 'Create your Twitter application' button.
-6. Navigate to the "Keys and Access Tokens" tab and copy the API_KEY and API_SECRET.
+6. Check the 'Developer Agreement' checkbox and click the 'Create your Twitter application' button.
```

Comment:
I usually keep the same number in lists, as MArkdown will replace the numbers automatically, and helps reducing changes when inserting new elements in between

Reply:
Yeah, the `5.` number was repeated and `prettier` fixed it this way. I agree that using `1.` is cleaner for the reasons you provided. Going to change it.